hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/TransactionChecker.java

```diff
@@ -221,7 +209,11 @@ public TransactionInfo check(@NonNull final Transaction tx, @Nullable Bytes seri
         final SignatureMap signatureMap;
         if (tx.hasBody()) {
             txBody = tx.bodyOrThrow();
-            bodyBytes = TransactionBody.PROTOBUF.toBytes(txBody);
+            try {
+                bodyBytes = serializedTx == null ? Bytes.EMPTY : ProtobufUtils.extractBodyBytes(serializedTx);
```

Comment:
If `serializedTx == null` don't we need to throw here `INVALID_TRANSACTION` ?

Reply:
Good catch! I removed the last place that called this method with `serializedTx=null`. I will fix it now.

Reply:
Because we have `txBody` here we don't need to throw here I think.

Reply:
Once we remove that we can make `TransactionInfo.serializedTransaction` to `@NonNull`

Reply:
Fixed